src/bin/cli/assets.js

```diff
@@ -0,0 +1,321 @@
+        await selectWitnetArtifacts(registry, assets, args, "  ", !flags?.all)
+    ).sort(([a,], [b,]) => { if (a < b) return -1; else if (a > b) return 1; else return 0; });
+    
+    if (selection.length > 0) {
+        for (const index in selection) {
+            const [ key, color, asset ] = selection[index]
+            
+            if (asset instanceof Witnet.Radon.RadonTemplate || asset instanceof Witnet.Radon.RadonModal) {
+                // todo: `lookupRadonTemplate(templates[network][key])` and compare with `asset`)
+                if (flags?.deploy /*&& deployables.templates[network][key] !== target*/) {
+                    console.info()
+                    const user = await prompt([{
+                        message: `Deploy ${asset.constructor.name}::${color(key)} ?`,
+                        name: "continue",
+                        type: "confirm",
+                        default: true,
+                    }])
+                    if (user.continue) {
+                        let target
+                        let gasUsed = BigInt(0)
+                        const template = await deployer.deployRadonTemplate(asset, {
+                            confirmations: 1,
+                            onVerifyRadonRetrieval: (hash) => {
+                                process.stdout.write(`  > Verifying parameterized data source => ${helpers.colors.gray(hash)} ... `)
+                            },
+                            onVerifyRadonRetrievalReceipt: (_hash, receipt) => {
+                                gasUsed += receipt?.gasUsed || BigInt(0)
+                                process.stdout.write(`${helpers.colors.lwhite("OK")}\n`)
+                            },
+                            onDeployRadonTemplate: (address) => {
+                                target = address
+                                process.stdout.write(`  > Replacing WitOracleRequestTemplate => ${helpers.colors.mblue(address)} ... `)
+                            },
+                            onDeployRadonTemplateReceipt: (receipt) => {
+                                gasUsed += receipt?.gasUsed || BigInt(0)
+                                process.stdout.write(`${helpers.colors.lwhite("OK")}\n`)
+                            }
+                        }).catch(err => { 
+                            process.stdout.write(`${helpers.colors.mred("FAIL")}`); 
+                            console.error(err)
+                            throw err
+                        })
+                        // process.stdout.write(`  > WitOracleRequestTemplate address: ${helpers.colors.mcyan(target)}\n`)
+                        deployables.templates[network][key] = template.address
+                        helpers.saveWitnetJsonFiles({ templates: deployables.templates })
+
+                        const artifact = await witOracle.getWitOracleRequestTemplateAt(target)
+                        for ([ sample, args ] of Object.entries(asset?.samples)) {
```

Comment:
We have to define variables in a `for...of
````suggestion
                        for (const [ sample, args ] of Object.entries(asset?.samples)) {
```